README.md

```diff
@@ -236,6 +236,11 @@ To run file of unit tests, use form:
 python3 -m unittest discover -p test_profile_builder.py
 ```
 
+To run a file with Pytest use:
+```
+pytest data_profiler/tests/data_readers/test_csv_data.py -v
```

Comment:
why do we need pytest here? if so, we have to make it for other test use cases, e.g., all tests

Reply:
Just an example of an alternative method. Pytest is part of the requirements-test.txt